TestPlaybooks/playbook-TestIsMaliciousIndicatorFound.yml

```diff
@@ -0,0 +1,413 @@
+      expectedValue: {}
+      fields: {}
+      path:
+        simple: xxx
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 1420
+        }
+      }
+  "6":
+    id: "6"
+    taskid: b03f84a8-b6ec-4a7c-86bb-9ac8d793a02e
+    type: condition
+    task:
+      id: b03f84a8-b6ec-4a7c-86bb-9ac8d793a02e
+      version: -1
+      name: IsMaliciuosFound
+      scriptName: IsMaliciousIndicatorFound
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "no":
+      - "5"
+      "yes":
+      - "13"
+    scriptarguments:
+      includeSuspicious: {}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 1245
+        }
+      }
+  "7":
+    id: "7"
+    taskid: c0e11d78-b854-49e4-8450-3eaf435408f1
+    type: regular
+    task:
+      id: c0e11d78-b854-49e4-8450-3eaf435408f1
+      version: -1
+      name: Create suspiciuos indicator
+      script: Builtin|||createNewIndicator
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "8"
+    scriptarguments:
+      internal: {}
+      merge: {}
+      note: {}
+      relatedIncidents:
+        simple: ${incident.id}
+      reputation:
+        simple: Suspicious
+      shorttextfield: {}
+      source: {}
+      sourceTimeStamp: {}
+      tags: {}
+      tmp: {}
+      type:
+        simple: Email
+      value:
+        simple: yulu@demisto.com
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 545
+        }
+      }
+  "8":
+    id: "8"
+    taskid: c6b5e42f-3366-46ec-8df3-8146ef0c33a9
+    type: condition
+    task:
+      id: c6b5e42f-3366-46ec-8df3-8146ef0c33a9
+      version: -1
+      name: IsMaliciuosFound-no-suspicious
+      scriptName: IsMaliciousIndicatorFound
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "no":
+      - "11"
+      "yes":
+      - "10"
+    scriptarguments:
+      includeSuspicious: {}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 720
+        }
+      }
+  "10":
+    id: "10"
+    taskid: 6f22921c-07e1-4315-83ab-5cbb9013a6c2
+    type: regular
+    task:
+      id: 6f22921c-07e1-4315-83ab-5cbb9013a6c2
+      version: -1
+      name: Fail
+      description: |-
+        Verifies path in context:
+        - Verifies path existence
+        - If matching object is an array: verify fields exists in each of the objects in the array
+        - If matching object is not an array: verify fields exists in matching object
+        - if 'expectedValue' is given: ensure that the given value is equal to the context path
+      scriptName: VerifyContext
+      type: regular
+      iscommand: false
+      brand: ""
+    scriptarguments:
+      expectedValue: {}
+      fields: {}
+      path:
+        simple: xxx
```

Comment:
I would change text here to imply which task failed (something like "Did not found suspicious indicator but it exists..")